src/oauth2/service.js

```diff
@@ -1,302 +1,325 @@
-var Service = require('../service');
+var AccessTokenRequest =
+require('../../oicMsg/oauth2/init.js').AccessTokenRequest;
+var AccessTokenResponse =
+require('../../oicMsg/oauth2/init.js').AccessTokenResponse;
+var AuthorizationRequest =
+require('../../oicMsg/oauth2/init.js').AuthorizationRequest;
+var oauth2 = require('../../oicMsg/oauth2/init.js');
+var Service = require('../service.js');
+var TokenErrorResponse =
+    require('../../oicMsg/oauth2/init.js').TokenErrorResponse;
 
-function postXParseResponse(resp, cliInfo, state){
-    state = state || '';
-    if (resp instanceof (AuthorizationResponse, AccessTokenResponse)){
-        cliInfo.stateDb.addMessageInfo(resp, state);
-    }
+function postXParseResponse(resp, cliInfo, state) {
+  state = state || '';
+  cliInfo.stateDb.addResponse(resp, state);
 };
```

Comment:
Remove trailing ; here and below when declaring functions (not variables or properties).